presentation/src/main/java/com/android254/presentation/login/view/LoginScreen.kt

```diff
@@ -15,20 +15,162 @@
  */
 package com.android254.presentation.login.view
 
-import androidx.compose.material3.Text
+import androidx.compose.animation.rememberSplineBasedDecay
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.*
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextDecoration
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.rememberNavController
+import com.android254.presentation.R
+import com.android254.presentation.common.components.DroidConTextField
+import com.android254.presentation.common.components.GoogleAuthButton
+import com.android254.presentation.common.navigation.Screens
 import com.android254.presentation.common.theme.DroidconKE2022Theme
+import com.android254.presentation.common.theme.Montserrat
 
 @Composable
-fun LoginScreen() {
-    Text(text = "Login Screen")
+fun LoginScreen(darkTheme: Boolean = isSystemInDarkTheme(), navController: NavHostController) {
+    val decayAnimationSpec = rememberSplineBasedDecay<Float>()
+    val scrollBehavior = TopAppBarDefaults.exitUntilCollapsedScrollBehavior(
+        decayAnimationSpec,
+        rememberTopAppBarScrollState()
+    )
+
+    Box() {
```

Comment:
Remove this since it's not being used we can have a scaffold only

Reply:
I suppose you are talking of line 48 - 52, right?

Reply:
The Box composable

Reply:
We need that so we can get the background image to show up behind the LargeTopAppBar.